x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/status_action.tsx

```diff
@@ -165,6 +165,16 @@ export const EndpointStatusActionResult = memo<
         ),
         description: <ConsoleCodeBlock>{agentStatus()}</ConsoleCodeBlock>,
       },
+      {
+        title: (
+          <ConsoleCodeBlock>
+            {i18n.translate('xpack.securitySolution.endpointResponseActions.status.agentPlatform', {
+              defaultMessage: 'Agent platform',
+            })}
+          </ConsoleCodeBlock>
+        ),
+        description: <ConsoleCodeBlock>{endpointDetails.metadata.host.os.family}</ConsoleCodeBlock>,
```

Comment:
Showing just the family is not very helpful (IMO) in the `status` command - especially since we already have the icon. In the Issue I had suggested you show the `.host.os.full` instead.